File: app/src/main/java/com/noti/sns/utility/BtnPress.java
package com.noti.sns.utility;

import android.app.Activity;
import android.os.Build;
import android.view.MotionEvent;
import android.widget.ImageView;
import android.widget.TextView;

import com.noti.sns.R;

public class BtnPress extends Activity {
	public static Boolean bigBTN(MotionEvent e, ImageView btn) {
		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {

			switch (e.getAction()) {
				case MotionEvent.ACTION_DOWN:
					btn.setBackgroundResource(R.drawable.btn_design_pressed);
					btn.setElevation(0F);
					switch (btn.getId()) {
						case R.id.login_btn:
							btn.setImageResource(R.drawable.ic_arrow_white_24dp);
							break;
						case R.id.register_confirm:
						case R.id.btnAlam:
							btn.setImageResource(R.drawable.ic_check_black_24dp);
							break;
					}
					break;
				case MotionEvent.ACTION_UP:
					btn.setBackgroundResource(R.drawable.btn_design);
					btn.setElevation(5F);
					switch (btn.getId()) {
						case R.id.login_btn:
							btn.setImageResource(R.drawable.ic_arrow_forward_24dp);
							break;
						case R.id.register_confirm:
						case R.id.btnAlam:
							btn.setImageResource(R.drawable.ic_check_blue_24dp);
							break;
					}
			}
		}
		return false;
	}

	public static Boolean addBTN(MotionEvent e, ImageView btn) {
		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
			if (e.getAction() == MotionEvent.ACTION_DOWN) {
				btn.setBackgroundResource(R.drawable.btn_design);
				btn.setElevation(0F);
				if (btn.getId() == R.id.btnAlam) btn.setImageResource(R.drawable.ic_add_black_24dp);
			} else if (e.getAction() == MotionEvent.ACTION_UP) {
				btn.setBackgroundResource(R.drawable.btn_design_pressed);
				btn.setElevation(5F);
				if (btn.getId() == R.id.btnAlam) btn.setImageResource(R.drawable.ic_add_white);
			}
		}
		return false;
	}

	public static Boolean smallBTN(MotionEvent e, ImageView btn) {
		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
			switch (e.getAction()) {
				case MotionEvent.ACTION_DOWN:
					btn.setBackgroundResource(R.drawable.btn_design2_pressed);
					switch (btn.getId()) {
						case R.id.login_btn:
							btn.setImageResource(R.drawable.ic_person_add_blue_24dp);
							break;
						case R.id.register_confirm:
							btn.setImageResource(R.drawable.ic_lock_blue_24dp);
							break;
					}
					break;
				case MotionEvent.ACTION_UP:
					btn.setBackgroundResource(R.drawable.btn_design2);
					switch (btn.getId()) {
						case R.id.login_btn:
							btn.setImageResource(R.drawable.ic_person_add_black_24dp);
							break;
						case R.id.register_confirm:
							btn.setImageResource(R.drawable.ic_lock_black_24dp);
							break;
					}
			}
		}
		return false;
	}

	public static Boolean pinBTN(MotionEvent e, TextView btn) {
		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {

			switch (e.getAction()) {
				case MotionEvent.ACTION_DOWN:
					btn.setBackgroundResource(R.drawable.btn_pin_pressed);
					break;
				case MotionEvent.ACTION_UP:
					btn.setBackgroundResource(R.drawable.btn_pin);
					break;
			}
		}
		return false;
	}

}

File: app/src/main/java/com/noti/sns/Btn_press.kt
package com.noti.sns

import android.app.Activity
import android.os.Build
import android.view.MotionEvent
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView


class Btn_press : Activity(){

    fun bigBTN(e : MotionEvent,btn : ImageView): Boolean {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {

            when (e.action) {
                MotionEvent.ACTION_DOWN -> {
                    btn.setBackgroundResource(R.drawable.btn_design_pressed)
                    btn.elevation = 0F
                    when (btn.id) {
                        R.id.login_btn ->
                            btn.setImageResource(R.drawable.ic_arrow_white_24dp)
                        R.id.register_confirm ->
                            btn.setImageResource(R.drawable.ic_check_black_24dp)
                    }
                }
                MotionEvent.ACTION_UP -> {
                    btn.setBackgroundResource(R.drawable.btn_design)
                    btn.elevation = 5F
                    when (btn.id) {
                        R.id.login_btn ->
                            btn.setImageResource(R.drawable.ic_arrow_forward_24dp)
                        R.id.register_confirm ->
                            btn.setImageResource(R.drawable.ic_check_blue_24dp)
                    }

                }
            }
        }
        return false
    }
    fun smallBTN(e : MotionEvent,btn : ImageView): Boolean {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {

            when (e.action) {
                MotionEvent.ACTION_DOWN -> {
                    btn.setBackgroundResource(R.drawable.btn_design2_pressed)
                    when (btn.id) {
                        R.id.register_btn ->
                            btn.setImageResource(R.drawable.ic_person_add_blue_24dp)
                        R.id.passwd_btn ->
                            btn.setImageResource(R.drawable.ic_lock_blue_24dp)
                    }
                }
                MotionEvent.ACTION_UP -> {
                    btn.setBackgroundResource(R.drawable.btn_design2)
                    when (btn.id) {
                        R.id.register_btn ->
                            btn.setImageResource(R.drawable.ic_person_add_black_24dp)
                        R.id.passwd_btn ->
                            btn.setImageResource(R.drawable.ic_lock_black_24dp)
                    }

                }
            }
        }
        return false
    }

    fun pinBTN(e : MotionEvent,btn : TextView): Boolean {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {

            when (e.action) {
                MotionEvent.ACTION_DOWN -> {
                    btn.setBackgroundResource(R.drawable.btn_pin_pressed)

                }
                MotionEvent.ACTION_UP -> {
                    btn.setBackgroundResource(R.drawable.btn_pin)


                }
            }
        }
        return false
    }

}
